draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1873,6 +1873,10 @@ The policy can also define limits on cookie size, cookie expiry (see
 {{attribute-expires}} and {{attribute-max-age}}), and the number of cookies per
 domain or in total.
 
+The recomended cookie expiry upper limit is 400 days, but this can be lower to
+reflect shorter browser data retention timelines or higher to support different
+use cases (e.g., machine-to-machine communication over HTTP).
```

Comment:
Style nit: "this can be" sounds a little strange to me. Perhaps:

```suggestion
The recomended cookie expiry upper limit is 400 days. User agents may set
a lower limit to enforce shorter data retention timelines, or set the limit higher
to support longer retention when appropriate (e.g., machine-to-machine
communication over HTTP).
```

I also wonder whether "machine-to-machine" should be more approachably framed as "server-to-server"? I'm not sure what the scenarios in question actually were...